packages/tools/locales/ca.po

```diff
@@ -4454,325 +4420,3 @@ msgstr "Ampliar"
-
-#~ msgid "Cannot move tag to this location."
-#~ msgstr "No es pot moure l'etiqueta a aquesta ubicació."
-
-#~ msgid "Tag name cannot start or end with a `/`."
-#~ msgstr "El nom de l'etiqueta no pot començar o acabar amb '/'."
-
-#~ msgid "Tag name cannot contain `//`."
-#~ msgstr "El nom de l'etiqueta no pot tenir '//'."
-
-#~ msgid "Synchronize"
-#~ msgstr "Sincronitza"
-
-#~ msgid "Json Export Directory"
-#~ msgstr "Directori d'exportació de JSON"
-
-#~ msgid ""
-#~ "This item is currently encrypted: %s \"%s\". Please wait for all items to "
-#~ "be decrypted and try again."
-#~ msgstr ""
-#~ "L'element ara mateix és xifrat: %s «%s». Espereu que es desxifrin tots "
-#~ "els elements i torneu-ho a intentar."
-
-#~ msgid "Add or remove tags"
-#~ msgstr "Afegeix o suprimeix etiquetes"
-
-#, fuzzy
-#~ msgid "Resources"
-#~ msgstr "Recursos: %d."
-
-#~ msgid "Global zoom percentage"
-#~ msgstr "Percentatge de zoom global"
-
-#~ msgid "Synchronisation is already in progress. State: %s"
-#~ msgstr "La sincronització ja és en procés. Estat: %s"
-
-#, fuzzy
-#~ msgid "Confirm master password:"
-#~ msgstr "Introduïu una contrasenya mestra:"
-
-#, fuzzy
-#~ msgid "Confirm password"
-#~ msgstr "Introduïu una contrasenya mestra:"
-
-#, fuzzy
-#~ msgid "Missing required argument: note"
-#~ msgstr "Manca un argument requerit: %s"
-
-#~ msgid "Synchronisation status"
-#~ msgstr "Estat de la sincronització"
-
-#~ msgid "General Options"
-#~ msgstr "Opcions generals"
-
-#~ msgid "Encryption options"
-#~ msgstr "Opcions del xifratge"
-
-#~ msgid "Encryption Options"
-#~ msgstr "Opcions de xifratge"
-
-#~ msgid "Clipper Options"
-#~ msgstr "Opcions del desa-retalls"
-
-#~ msgid "Separate each tag by a comma."
-#~ msgstr "Separeu les etiquetes amb comes."
-
-#~ msgid "State: %s."
-#~ msgstr "Estat: %s."
-
-#~ msgid "A notebook with this title already exists: \"%s\""
-#~ msgstr "Ja existeix un bloc de notes amb aquest títol: «%s»"
-
-#~ msgid ""
-#~ "The path to synchronise with when file system synchronisation is enabled. "
-#~ "See `sync.target`."
-#~ msgstr ""
-#~ "El camí on sincronitzar en activar la sincronització del sistema. Vegeu "
-#~ "«sync.target»."
-
-#~ msgid "Cancel synchronisation"
-#~ msgstr "Cancel·la la sincronització"
-
-#~ msgid "Hide metadata"
-#~ msgstr "Amaga les metadades"
-
-#~ msgid "Show metadata"
-#~ msgstr "Mostra les metadades"
-
-#~ msgid "Delete notebook"
-#~ msgstr "Suprimeix el bloc de notes"
-
-#~ msgid ""
-#~ "Click on the (+) button to create a new note or notebook. Click on the "
-#~ "side menu to access your existing notebooks."
-#~ msgstr ""
-#~ "Feu clic en el botó (+) per a crear una nota o bloc de notes. Feu clic en "
-#~ "el menú lateral per a accedir als blocs de notes existents."
-
-#~ msgid ""
-#~ "You currently have no notebook. Create one by clicking on (+) button."
-#~ msgstr ""
-#~ "Ara mateix no teniu cap bloc de notes. Creeu-ne un fent clic al botó (+)."
-
-#~ msgid "Welcome"
-#~ msgstr "Benvingut"
```

Comment:
> Did you use Poedit to make the changes? Just as an info, you are not supposed to manually edit the .po files.

No, I did not use PoEdit (is not open source).

> Please do not delete these.

(not challenging, just trying to understand): any reason to keep obsolete strings in the file?

Reply:
We rather keep obsolete strings since they might come up again in the fuure, which means we will have a proper translation. That is if the translation of the obsolete string was correct in the first place.

These strings would not have shown up in Poedit had you used it.

Please note that Poedit is not open source but free to use. If you want to do translations you will have to use it.
We ran into so many issues when people manually edited files that Poedit is now a requirement. If you use Linux there are other alternatives. As long as the app does some sort of verification that the file is valid, it doesn't really matter which app is used.

Reply:
Yes, I'm familiar with issues that can happen when editing po files manually.

I've added the obsolete strings back. Thanks for the info!

Reply:
The basic version of poEdit (which is all that's needed) is actually open source.

Reply:
it is free (as in beer), but is no longer open source.... :(

In any case, the file is updated and validated (I use another PO file manager, and I'm familiar with gettext tools).

It's just that the program I use deletes (by default) obsolete strings.